dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesRequestData.java

```diff
@@ -70,8 +70,12 @@ public WesRequestData(String url, String awsAccessKey, String awsSecretKey, Stri
 
         // The credentials that were passed in are null, so we are assuming they are within an authorized environment (such as an EC2 instance)
```

Comment:
This comment no longer seems applicable here. Can you please move it to where it is applicable, if I'm reading this right.

Reply:
Reworded: https://github.com/dockstore/dockstore-cli/pull/99/commits/e4fe3cc94e2f5bf1587faa9f23acc57afd92b334